examples/quickstart-sklearn-tabular/README.md

```diff
@@ -4,80 +4,62 @@ dataset: [Iris]
 ```
 
-This will create a new directory called `quickstart-sklearn-tabular` containing the following files:
+This will create a new directory called `quickstart-sklearn-tabular` with the following structure:
 
 ```shell
--- pyproject.toml
--- requirements.txt
--- client.py
--- server.py
--- utils.py
--- README.md
+quickstart-sklearn-tabular
+├── sklearnexample
+│   ├── __init__.py
+│   ├── client_app.py   # Defines your ClientApp
+│   ├── server_app.py   # Defines your ServerApp
+│   └── task.py         # Defines your model, training and data loading
+├── pyproject.toml      # Project metadata like dependencies and configs
+└── README.md
 ```
 
-### Installing Dependencies
-
-Project dependencies (such as `scikit-learn` and `flwr`) are defined in `pyproject.toml` and `requirements.txt`. We recommend [Poetry](https://python-poetry.org/docs/) to install those dependencies and manage your virtual environment ([Poetry installation](https://python-poetry.org/docs/#installation)) or [pip](https://pip.pypa.io/en/latest/development/), but feel free to use a different way of installing dependencies and managing virtual environments if you have other preferences.
+### Install dependencies and project
 
-#### Poetry
+Install the dependencies defined in `pyproject.toml` as well as the `sklearnexample` package.
 
-```shell
-poetry install
-poetry shell
+```bash
+pip install -e .
 ```
 
-Poetry will install all your dependencies in a newly created virtual environment. To verify that everything works correctly you can run the following command:
+## Run the project
 
-```shell
-poetry run python3 -c "import flwr"
-```
+You can run your Flower project in both _simulation_ and _deployment_ mode without making changes to the code. If you are starting with Flower, we recommend you using the _simulation_ mode as it requires fewer components to be launched manually. By default, `flwr run` will make use of the Simulation Engine.
 
-If you don't see any errors you're good to go!
+### Run with the Simulation Engine
 
-#### pip
-
-Write the command below in your terminal to install the dependencies according to the configuration file requirements.txt.
-
-```shell
-pip install -r requirements.txt
+```bash
+flwr run .
 ```
 
-## Run Federated Learning with scikit-learn and Flower
-
-Afterwards you are ready to start the Flower server as well as the clients. You can simply start the server in a terminal as follows:
+You can also override some of the settings for your `ClientApp` and `ServerApp` defined in `pyproject.toml`. For example:
 
-```shell
-poetry run python3 server.py
-```
-
-Now you are ready to start the Flower clients which will participate in the learning. To do so simply open two more terminals and run the following command in each:
-
-```shell
-poetry run python3 client.py --partition-id 0 # partition-id should be any of {0,1,2}
+```bash
+flwr run . --run-config penalty="l1"
```

Comment:
```suggestion
flwr run . --run-config num-server-rounds =10
```

Reply:
it seems "l1" penalty is not supported, so i'd suggest showing the overriding functionality with another setting.

```
ValueError: Solver lbfgs supports only 'l2' or None penalties, got l1 penalty.
```

Reply:
I've changed the solver to "saga" now which supports also `l1`.

Though I cannot run the 
`flwr run . --run-config penalty="l1"`

`CalledProcessError: Command '['flower-simulation', '--app', '.', '--num-supernodes', '3', '--run-config', 'penalty=l1']' returned non-zero exit status 1.`